app/i18n/raw-i18n/translations/qu.json

```diff
@@ -723,7 +723,7 @@
         "learn": "Ñuqaqa kaniyta mana yachanakuyta munanki, hinaspak kaniyta kawsayniyta kaniy, ch'usaq kaniyta...",
         "learnToEarn": "Yachanakuyta kaniyta kawsayniyta kaniy"
     },
-    "PhoneInputScreen": {
+    "PhoneLoginInitiateScreen": {
```

Comment:
These manual fixes will get overwritten by transifex the next time we sync the languages. Perhaps there is a control in transifex to just rename a field.